yahoo.com

```diff
@@ -23,30 +22,36 @@
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
```

Comment:
Is OAuth2 required yet?
Right now, do end users have to manually change settings (that is, either enable "insecure apps" manually, or create a specific IMAP password) in order to use Yahoo mail in Thunderbird?

Reply:
I didn't understand this question. Yahoo! users on OAuth2 enabled clients can: (a) use OAuth2 for authenticating, (b) Use an app or IMAP specific password, or (c) Enable "insecure" apps and use their account password.